app/views/practices/complexity.html.erb

```diff
@@ -41,13 +41,13 @@
                     </p>
                     <fieldset class="usa-fieldset">
                         <div class="grid-row grid-gap margin-top-105 margin-bottom-3">
-                            <% Department.order("CASE WHEN name = 'None' THEN 1 WHEN name = 'All' THEN 2 ELSE 3 END, name").each do |department| %>
+                            <% Department.order("CASE WHEN name = 'None' THEN 1 WHEN name = 'All departments equally - not a search differentiator' THEN 2 ELSE 3 END, name").each do |department| %>
                               <% if department.name == 'None' %>
                                 <div class='usa-checkbox grid-col-6'>
                                   <input type='checkbox' class='usa-checkbox__input no-department-input' id="practice_department_<%= @no_departments.id %>" name="practice[department][<%= @no_departments.id %>][value]" <%= !@practice.departments.find_by(name: 'None').nil? ? "checked= 'true'" : "" %>>
                                   <label class='usa-checkbox__label' for="practice_department_<%= @no_departments.id %>">None</label>
                                 </div>
-                              <% elsif department.name == 'All' %>
+                              <% elsif department.name == 'All departments equally - not a search differentiator' %>
```

Comment:
wondering if we should do a unique short name or something here

Reply:
gonna merge this for now.
i'll think about this more when it bites us again :smile:

Reply:
Can I go in and write a task to assign a short name to this department?

Reply:
i think i also want `short_name` to be unique

Reply:
we decided to wait on this :)